libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -101,8 +101,8 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
 #if GPS_MAX_RECEIVERS > 1
     // @Param: AUTO_SWITCH
     // @DisplayName: Automatic Switchover Setting
-    // @Description: Automatic switchover to GPS reporting best lock
-    // @Values: 0:Disabled,1:UseBest,2:Blend,3:UseSecond
+    // @Description: Automatic switchover to GPS reporting best lock, 1:UseBest selects the GPS with highest status, if both are equal the GPS with highest satellites is used. 4:Use primary (as set with  GPS_PRIMARY) conservative uses the GPS with the highest status, if both are equal the primary will be used until it has 4 satellites fewer, as soon primary has more it will be used again
+    // @Values: 0:Disabled,1:UseBest,2:Blend,4:Use primary conservative switch
```

Comment:
add "use primary as long as it has 3D fix"

Reply:
changed to this